.github/workflows/e2e_windows.yaml

```diff
@@ -19,6 +19,18 @@ jobs:
   e2e-test-windows:
     runs-on:
       labels: amplify-flutter_windows-latest_8-core
+    strategy:
+      # Allows other matrix items to run if one fails
+      fail-fast: false
+      matrix:
+        channel:
+          - "beta"
+          - "stable"
```

Comment:
You are correct. We are not consistent on including or excluding them across our actions. Unless there is a good reason to remove them I will keep them to avoid running all the tests again.

Reply:
I ended up changing this since I had to push another change